### PR TITLE
Added attributes for administration pages

### PR DESCRIPTION
**Pull request by ghost**
Ported ['cups']['admin']['auth_read'] and ['cups']['admin']['acl'] from debian-8 to default template
Added attribute ['cups']['admin']['require_encryption'] to debian-8 and default template
Added documentation for this attributes to README.md
Pushed version in metadata.rb

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 source_url       'https://github.com/artem-sidorenko/chef-cups'
 issues_url       'https://github.com/artem-sidorenko/chef-cups/issues'
-version          '0.12.0'
+version          '0.13.0'
```
Comment (artem-sidorenko) [inline metadata.rb]:
```suggestion
version          '0.12.0'
```

@olqs may I ask you to remove the version bump? I'll do it by myself during the release procedure
```suggestion
version          '0.13.0'
```

**Comment by artem-sidorenko** [comment]
@olqs thanks for this PR! Sorry, I somehow totally missed it, maybe because of long parental leaves this year.

Can you please remove the version bump? I'll have a deeper look to this PR today/in the next days. Thanks!

**Comment by olqs** [comment]
@artem-sidorenko 
i have removed the version bump. I hope i did it right. It's was the first time i made a pull request.

**Comment by artem-sidorenko** [comment]
@olqs many thanks for this PR!

**Comment by artem-sidorenko** [comment]
@olqs 0.13.0 is released

**Pull request merged**